### PR TITLE
[IMP] all modules: Unifomize Modals

### PR DESCRIPTION
**Pull request by rba-odoo**
Task: https://www.odoo.com/web#id=19645&view_type=form&model=project.task&menu_id=3942&action=327
Pad: https://pad.odoo.com/p/r.Y1fLImzP4ay8emS2

### CODE REVIEW

File: addons/web/static/src/css/base.css [hunk 1]
```diff
@@ -439,7 +439,7 @@
   max-width: 180px;
   max-height: 180px;
 }
-.openerp .oe_button.oe_link {
```
Comment (hmo-odoo) [inline addons/web/static/src/css/base.css]: can you change sass file also recording this changes ?

Comment (rba-odoo) [inline addons/web/static/src/css/base.css]: Done!

**Comment by rba-odoo** [comment]
Work continues https://github.com/odoo/odoo/pull/8102

**Comment by rba-odoo** [comment]
Merged at https://github.com/odoo/odoo/commit/393f136509622f40f452ce9848d618977d5cd6e9

**Pull request closed without merging**